client/src/pages/LandingPage/LandingPage.js

```diff
@@ -20,15 +22,34 @@ class LandingPage extends React.Component {
 
     handleSubmit = event => {
         event.preventDefault();
-        // if (!this.state.password || !this.state.username){
-        //     alert('Please fill out all manditory fields.')
-        // }
-        // else (
-            API.Login({
-                username: this.state.username,
-                password: this.state.password,
-            })
-        // )
+
+        API.checkUsername(
+            this.state.username
+        ).then(res => {
+
+            if(res.data.password === this.state.password){
+                this.setState({
+                    success: true,
+                    id: res.data._id
+                })
+                window.location.href = `/profile/${this.state.id}`;
+            } else {
+                this.handleInvalidLogin();
+            }
+        })
+        .catch(err => {
+            this.handleInvalidLogin();
+            console.log(err)
+            return err
+        });
+    }
+
+    handleInvalidLogin = () => {
+        this.setState({
+            username: '',
```

Comment:
We might want to consider not blanking out the username input so the user can re-enter the password if it fails.